avocado/core/loader.py

```diff
@@ -615,6 +617,9 @@ def _find_avocado_tests(self, path):
         # The resulting test classes
         result = {}
 
+        if os.path.isdir(path):
+            path = os.path.join(path, "__init__.py")
```

Comment:
not worth changing it now, but if it comes to sending next version, you should split this change into a separate commit (it adds different functionality)